pages/datasets/[did].vue

```diff
@@ -0,0 +1,210 @@
+      >
+        <Breadcrumb>
+          <BreadcrumbItem
+            to="/"
+            :external="true"
+          >
+            {{ $t('Accueil') }}
+          </BreadcrumbItem>
+          <BreadcrumbItem to="/datasets">
+            {{ $t('Jeux de données') }}
+          </BreadcrumbItem>
+          <BreadcrumbItem>
+            {{ dataset.title }}
+          </BreadcrumbItem>
+        </Breadcrumb>
+        <div class="flex flex-wrap gap-2.5 md:max-w-6/12">
+          <FollowButton
+            v-if="dataset"
+            :url="`/api/1/datasets/${dataset.id}/followers/`"
+          />
+          <ReportModal
+            v-if="!isOrganizationCertified(dataset.organization)"
+            :subject="{ id: dataset.id, class: 'Dataset' }"
+          />
+          <EditButton
+            v-if="isAdmin(me)"
+            :id="dataset.id"
+            type="datasets"
+          />
+        </div>
+      </div>
+    </div>
+    <LoadingBlock
+      v-if="dataset"
+      :status
+    >
+      <div class="space-y-8">
+        <div class="container pt-3 min-h-32">
+          <h1 class="text-2xl text-gray-title mb-6 font-extrabold">
+            {{ dataset.title }}
+          </h1>
+          <div class="flex flex-col md:space-x-10 md:flex-row">
+            <div class="flex-1 overflow-x-hidden">
+              <ReadMore class="">
+                <MarkdownViewer
+                  :content="dataset.description"
+                  :min-heading="3"
+                />
+              </ReadMore>
+            </div>
+            <dl class="pl-0 w-full shrink-0 md:w-[384px] space-y-3">
+              <div class="space-y-1">
+                <dt class="text-gray-plain font-bold mb-0 pb-0">
+                  {{ $t('Producer') }}
+                </dt>
+                <dd class="p-0 text-sm">
+                  <OrganizationOwner
+                    v-if="dataset.organization"
+                    :organization="dataset.organization"
+                  />
+                  <AvatarWithName
+                    v-if="dataset.owner"
+                    :size="32"
+                    :user="dataset.owner"
+                    class="space-x-2"
+                  />
+                  <SimpleBanner
+                    v-if="dataset.owner"
+                    type="warning"
+                    class="text-sm"
+                  >
+                    {{ $t("Ce jeu de données a été publié à l'initiative et sous la responsabilité de {author}.", { author: `${dataset.owner.first_name} ${dataset.owner.last_name}` }) }}
+                  </SimpleBanner>
+                </dd>
+              </div>
+
+              <div
+                v-if="dataset.contact_points.length"
+                class="space-y-1"
+              >
+                <dt class="text-gray-plain font-bold mb-0 pb-0">
+                  {{ $t('Contact') }}
+                </dt>
+                <dd class="p-0 text-sm">
+                  <ContactPoint
```

Comment:
I think the target blank is shown even if no link found (ex: http://dev.local:3000/fr/datasets/repartition-des-habitats-rapportes-sous-la-directive-habitats-faune-flore-en-2013-dhff-2013/)